specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/runNotebook.json

```diff
@@ -0,0 +1,359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-12-01",
```

Comment:
>2020-12-01 [](http://example.com/codeflow?start=16&length=10)

The API version should be consistent with folder name. It should be 2021-06-01-preview.

By the way, I don't  see `api-version` is used anywhere. This might be incorrect.

Reply:
Like [LivyApi](https://github.com/Azure/azure-rest-api-specs/blob/37c4ff1612668f5acec62dea729ca3a66b591d7f/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/sparkJob.json), we put API version into URL path instead of query string.